### PR TITLE
[ID] Add `osu!_Community_Meetings`

### PR DESCRIPTION
**Pull request by ZeroPyrozen**
Add osu! community meetings Indonesian translation
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/osu!_Community_Meetings/id.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+Saat ini, pertemuan komunitas osu! diketuai oleh [peppy](https://osu.ppy.sh/users/2), [Ephemeral](https://osu.ppy.sh/users/102335), dan [smoogipoo](https://osu.ppy.sh/users/1040328).
+
+Pertemuan komunitas osu! berikutnya akan diadakan pada **Minggu, 28 November 2021 pukul 01:00 UTC**. Pertemuan diadakan setiap dua minggu dengan pengingat yang diumumkan melalui [akun Twitter peppy](https://twitter.com/ppy) sehari sebelumnya.
+
+## Cara berpartisipasi
+
+Pertemuan komunitas osu! saat ini diadakan di Discord Stage pada [server Discord osu!dev](https://discord.gg/ppy) dan disiarkan pada [kanal Twitch peppy](https://www.twitch.tv/ppy). Setiap pertemuan saat ini dijadwalkan untuk berlangsung selama satu jam dan dimulai setiap hari Minggu pukul 01.00 UTC.
```
Comment (cl8n) [inline wiki/osu!_Community_Meetings/id.md:13]: typo in the meeting time, i think

Comment (Walavouchey) [reply wiki/osu!_Community_Meetings/id.md:13]: not all languages use a period between hours and minutes

Comment (cl8n) [reply wiki/osu!_Community_Meetings/id.md:13]: paragraph above this says `pukul 01:00 UTC` and here it's `pukul 01.00 UTC` so i comment. resolve if it's already correct though

Comment (ZeroPyrozen) [reply wiki/osu!_Community_Meetings/id.md:13]: ah we use dot to separate hour, minute, and second in time. [Pedoman Umum Ejaan Bahasa Indonesia (General Guidelines for Indonesian Spelling)](https://puebi.readthedocs.io/en/latest/tanda-baca/tanda-titik/#a-tanda-titik)

File: wiki/osu!_Community_Meetings/id.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Pertemuan komunitas osu!
+
+**Pertemuan komunitas osu!** adalah panel diskusi yang diadakan oleh [tim inti osu!](/wiki/People/The_Team) setiap dua minggu sekali. Tujuan utama dari pertemuan-pertemuan tersebut adalah untuk memberikan kesempatan untuk seluruh pihak untuk berbicara secara langsung dengan pengembang dan orang-orang yang bertanggung jawab dalam mengatur komunitas untuk mengangkat masalah-masalah untuk didiskusikan atau pertimbangan lebih lanjut.
```
Comment (wowcake) [inline wiki/osu!_Community_Meetings/id.md:3]:
```suggestion
**Pertemuan komunitas osu!** adalah panel diskusi yang diadakan oleh [tim inti osu!](/wiki/People/The_Team) setiap dua minggu sekali. Tujuan utama dari pertemuan-pertemuan tersebut adalah untuk memberikan kesempatan kepada seluruh pihak untuk berbicara secara langsung dengan pengembang dan orang-orang yang bertanggung jawab dalam mengatur komunitas untuk membahas tentang suatu hal atau pertimbangan lebih lanjut.
```

File: wiki/osu!_Community_Meetings/id.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+[Pada masa yang sangat lampau osu!](https://twitter.com/ppy/status/1169256824052170755), dulunya tim inti osu! rutin mengadakan pertemuan komunitas di mana seluruh orang dapat berbicara. Dengan ukuran permainan saat ini, hal ini tidak dapat dicapai, tetapi dengan menggunakan sistem antrian Discord yang dapat mengurutkan pertanyaan-pertanyaan yang dapat dibahas satu per satu. Jika kamu tertarik untuk bertanya, mohon pastikan kamu telah membaca FAQ di bawah ini untuk mengecek apakah pertanyaanmu sudah pernah ditanyakan dan dijawab!
```
Comment (wowcake) [inline wiki/osu!_Community_Meetings/id.md:5]:
```suggestion
[Pada masa yang sangat lampau osu!](https://twitter.com/ppy/status/1169256824052170755), dulunya tim inti osu! rutin mengadakan pertemuan komunitas di mana seluruh orang dapat berbicara. Dengan permainan yang mempunyai sebuah komunitas yang sangat besar, hal ini tidak dapat dicapai, tetapi dengan menggunakan sistem antrian Discord yang dapat mengurutkan pertanyaan-pertanyaan yang dapat dibahas satu per satu. Jika kamu tertarik untuk bertanya, mohon pastikan kamu telah membaca FAQ di bawah ini untuk mengecek apakah pertanyaanmu sudah pernah ditanyakan dan dijawab!

File: wiki/osu!_Community_Meetings/id.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+Saat ini, pertemuan komunitas osu! diketuai oleh [peppy](https://osu.ppy.sh/users/2), [Ephemeral](https://osu.ppy.sh/users/102335), dan [smoogipoo](https://osu.ppy.sh/users/1040328).
+
+Pertemuan komunitas osu! berikutnya akan diadakan pada **Minggu, 28 November 2021 pukul 01:00 UTC**. Pertemuan diadakan setiap dua minggu dengan pengingat yang diumumkan melalui [akun Twitter peppy](https://twitter.com/ppy) sehari sebelumnya.
```
Comment (wowcake) [inline wiki/osu!_Community_Meetings/id.md]:
```suggestion
Pertemuan komunitas osu! berikutnya akan diadakan pada **Minggu, 28 November 2021 pukul 01:00 UTC (08:00 WIB)**. Pertemuan diadakan setiap dua minggu dengan pengingat yang diumumkan melalui [akun Twitter peppy](https://twitter.com/ppy) sehari sebelumnya.

File: wiki/osu!_Community_Meetings/id.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+## Cara berpartisipasi
+
+Pertemuan komunitas osu! saat ini diadakan di Discord Stage pada [server Discord osu!dev](https://discord.gg/ppy) dan disiarkan pada [kanal Twitch peppy](https://www.twitch.tv/ppy). Setiap pertemuan saat ini dijadwalkan untuk berlangsung selama satu jam dan dimulai setiap hari Minggu pukul 01.00 UTC.
+
+Semua diperkenankan untuk berpartisipasi! Jika kamu punya pertanyaan atau sesuatu untuk disampaikan pada diskusi, kamu dapat mengangkat tanganmu di Discord Stage untuk masuk dalam antrian atau menuliskan pesan pada utas Discord terkait jika kamu tidak nyaman berbicara, yang akan dibacakan dan dijawab. Sebagai tambahan, sebelum bertanya, perhatikan pedoman berikut ini:
```
Comment (wowcake) [inline wiki/osu!_Community_Meetings/id.md:15]:
```suggestion
Semua orang diperkenankan untuk berpartisipasi! Jika kamu punya pertanyaan atau sesuatu untuk disampaikan pada diskusi, kamu dapat mengangkat tanganmu di Discord Stage untuk masuk dalam antrian atau menuliskan pesan pada utas Discord terkait jika kamu tidak nyaman berbicara, yang akan dibacakan dan dijawab. Sebagai tambahan, sebelum bertanya, perhatikan pedoman berikut ini:
```

File: wiki/osu!_Community_Meetings/id.md (hunk 4 above)
Comment (wowcake) [reply wiki/osu!_Community_Meetings/id.md]: not sure about this one, but does help people from misunderstanding the time

**Review by wowcake — CHANGES_REQUESTED** [review]

**Review by cl8n — APPROVED** [review]

**Review by wowcake — APPROVED** [review]
looks good to me!

**Pull request merged**